docker-compose-dev.yml

```diff
@@ -11,6 +11,8 @@ services:
       context: ./backend
       target: ${BUILD_TARGET:-dev}
     container_name: basegun-backend
+    command: uvicorn src.main:app --reload --host 0.0.0.0 --port 8000

```

Comment:
doesn't that duplicate the command ? It's already at the end of the Dockerfile, I don't understand why we need to put it here again

Reply:
Actually it is more practical to set the command here, or in the helm chart, it allow us to have a cleaner docker image which will be the same in each environment. Then we just need to set the command we want for each env, which is going to be different (exemple: with or without the `--reload`).